release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -869,6 +869,14 @@ Kubernetes 1.27 removed the following deprecated API, so you must migrate manife
 
 {product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for earlier setups.
 
+[id="ocp-4-14-future-deprecation"]
+=== Notice of future deprecation
+
+[id="ocp-4-14-future-deprecation-sdn"]
+==== Future deprecation of the OpenShift SDN network plugin
+
+Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be an option for new installations, but will continue to be supported in clusters upgrading to 4.15 and 4.16 from clusters previously installed with the OpenShift SDN network plugin. In a future release, targeting no earlier than 4.17, the OpenShift SDN network plugin will be deprecated and will no longer be supported.
```

Comment:
```suggestion
Targeting {product-title} 4.15, the OpenShift SDN CNI network plug-in will not be an option for new installations, but will continue to be supported in clusters upgrading to 4.15 and 4.16 from clusters previously installed with the OpenShift SDN network plug-in. In a future release, targeting no earlier than 4.17, the OpenShift SDN network plug-in will be deprecated and will no longer be supported.
```

Reply:
@tmalove Red Hat style uses https://redhat-documentation.github.io/supplementary-style-guide/#plugin, which supersedes the IBM guidance